### ISSUE TITLE
Ability to disable unused variables per function

### ISSUE THREAD

**Issue by satazor** [jshint/jshint#639]
``` js
/**
 * Emits an event.
 *
 * @param {String}   event   The event name
 * @param {...mixed} [$args] The arguments to pass to each listener
*/
emit: function (event, $args) {
    this._doSomething.apply(this, arguments);
},
```

I find  the warnings regarding unused variables very useful but I would like JSHint to ignore it in certain cases.
Above is a valid use case in which I want the arguments to be there for readability but they are not used.
I tried the inline `/*jshint unused:false*/` without success.

**Comment by ohmirocks** [comment]
HI,
Is this issue fixed. We are still having this problem in our CI builds when it tries to generate the checkstyle file

**Comment by jugglinmike** [comment]
@ohmirocks This issue was closed from commit 4ac15f6 , which was intended to implement the requested feature. If it is not working for you, we'd certainly appreciate an example which demonstrates your problem
